DMCompiler/DreamPath.cs

```diff
@@ -113,6 +113,16 @@ public enum PathType {
     public void SetFromString(string rawPath) {
         char pathTypeChar = rawPath[0];
         string[] tempElements = rawPath.Split('/', StringSplitOptions.RemoveEmptyEntries);
+                //operator/ and operator/= need special handling
+        if(rawPath.EndsWith("operator/"))
+            tempElements[^1] = "operator/";
+                //operator/ and operator/= need special handling
```

Comment:
Slight spacing here may make this easier to read
```suggestion
                // operator/ and operator/= need special handling
        if(rawPath.EndsWith("operator/"))
            tempElements[^1] = "operator/";
                // operator/ and operator/= need special handling
```